core/src/main/java/io/jenkins/pluginhealth/scoring/probes/Probe.java

```diff
@@ -45,22 +45,47 @@ public final ProbeResult apply(Plugin plugin, ProbeContext context) {
     }
 
     /**
-     * Perform the analyze on a plugin
+     * Perform the analyze on a plugin.
```

Comment:
```suggestion
     * Performs the analysis on a plugin.
```
The word `analyze` is giving a warning on IntelliJ, what do you think about replacing all its occurrences with `analysis`?